$MirOS: ports/misc/mc/patches/patch-lib_mc_menu,v 1.7 2006/02/21 15:12:13 tg Exp $
--- lib/mc.menu.orig	Tue Aug 17 08:30:54 2004
+++ lib/mc.menu	Mon Oct  2 23:39:36 2006
@@ -1,5 +1,42 @@
 shell_patterns=0
+= t r
+c       Diff current file to CVS
+	cvs -Rq diff -pdu %f 2>&1 | less
+
+= t r
+C       Diff current file to baseline CVS
+	cvs -Rq diff -pdur1.1.1 %f 2>&1 | less
+
+= t r
+e       Edit current revision diff against baserev
+	mksh /usr/src/scripts/editdiff %f
+
+= t r
+E       Edit current revision diff against any rev
+	mksh /usr/src/scripts/editdiff %f ask
+
+l       Show CVS log of file
+	cvs -Rq log %f | less
+
++ t r & ! t t
+u       Diff file against opposite
+	diff -pdu %f %D/%F 2>&1 | less
+
+= t d
+U       Diff dir against opposite
+	diff -pdu . %D 2>&1 | grep -v "^Only in " | less
+
+= t r
 + ! t t
+R       Reverse CVS problem
+	mv ".#%f".[0-9]* "%f"
+
+= t r
+P       Prepare for patching - copy to *.orig
+	test -e %f.orig && mv %f.orig %f.orig-$$
+	mv %f %f.orig && cp %f.orig %f
+
++ ! t t
 @       Do something on the current file
 	CMD=%{Enter command}
 	$CMD %f
@@ -21,7 +58,7 @@ shell_patterns=0
 
 =+ f \.1$ | f \.3$ | f \.4$ | f \.5$ | f \.6$ | f \.7$ | f \.8$ | f \.man$ & t r
 1       Display the file with roff -man
-	nroff -man %f | less
+	nrcon -man %f | less
 
 2       Call the info hypertext browser
         info
